deploy/templates/nstemplatetiers/advanced-code.yaml

```diff
@@ -15,6 +15,18 @@ objects:
     labels:
       provider: codeready-toolchain
     name: ${USERNAME}-code
+- apiVersion: v1
+  kind: RoleBinding
+  metadata:
+    labels:
+      provider: codeready-toolchain
+    name: codeready-toolchain-code
+    namespace: ${USERNAME}-code
+  roleRef:
+    name: admin
```

Comment:
May I know why `admin` and not `edit`? By giving the admin rights we also give the ability to delete the namespace, don't we?

Reply:
I think edit doesn’t allow creating/updating rbac. And it would be nice to allow advanced users to share the namespace. But allowing deleting namespaces is dangerous...
Let me double check the permissions.

Reply:
OK I checked it. "admin" role gives access to deleting namespaces and manipulating with rbac.
"edit" doesn't allow any of that.
I changed admin to edit for now. We can address the missing rbac thing for advanced tier later.
See https://github.com/codeready-toolchain/host-operator/pull/106/commits/d17c13c1e5c0d47270b843b45f7e33389089bfce

Reply:
BTW deleting a namespace should not be a big deal. The controller will recreate it. But anyway let's deal with that in a separate PR. For now we will use "edit" role for all tiers.